svc/ValidationService.js

```diff
@@ -0,0 +1,31 @@
+import {HoistService} from '@xh/hoist/core';
```

Comment:
Don't think this needs to be a service, if it has no entry points.  Why not do this setup within ValidationModel?  Can run code when the module loads (or lazily when first one is created, but I think simplest is better)